faq/index.html

```diff
@@ -54,8 +54,9 @@ <h6 class="arrow-r">Who’s using this and why?</h6>
                     <li class="submenu">
                         <h6 class="arrow-r">What programming languages are supported?</h6>
                         <div class="submenu-content">
-                            <p>C++, Java, Objective-C, Python, Ruby, Go, C#, Node.js.
-                            All of the above mentioned languages are now in GA and follow semantic versioning.</p>
+                            <p>C++, Java, Objective-C, Python, Ruby, Go, C#, Node.js are 
+                            in GA and follow semantic versioning.</p>
+                            <p>Dart support is in beta.</p>
```

Comment:
nit: move one line below? Android Java na iOS are also GA.

Reply:
The line following it strikes me as a little, shall we say, confusing...

How about just this:

```
                            <p>C++, Java (incl. Android), Objective-C (for iOS), Python, Ruby, Go, C#, Node.js are 
                            in GA and follow semantic versioning.</p>
                            <p>Dart support is in beta.</p>
```

Reply:
sg

Reply:
maybe I'd say "Java (incl. support for Android)"